### ISSUE TITLE
error: please complete registration using the osu! website

### ISSUE THREAD

**Issue by davidak** [ppy/osu#24498]
### Type

Other

### Bug description

I want to try out this game, so i installed it and registered an account ingame. When clicking "Register", it says in red text:

>please complete registration using the osu! website

but i see no way to do that.

I tried to login on

- https://osu.ppy.sh/
- https://lazer.ppy.sh/

but it says "Incorrect sign in".

![Screenshot from 2023-08-11 04-00-45](https://github.com/ppy/osu/assets/91113/7e87889e-96a3-426c-814f-752fce42103d)

I also did not get an e-mail to the used address.

### Screenshots or videos

![osu_2023-08-11_04-32-00](https://github.com/ppy/osu/assets/91113/b8864e64-ec76-4e2a-bd3b-536d974b7eda)

### Version

2023.803.0-lazer

### Logs

```
2023-08-11 01:56:48 [verbose]: Request to https://lazer.ppy.sh/users failed with System.Net.WebException: Forbidden.
2023-08-11 01:56:48 [verbose]: Response was: {"error":"please complete registration using the osu! website","url":"https:\/\/lazer.ppy.sh\/users\/create"}
```

[database.log](https://github.com/ppy/osu/files/12318229/database.log)
[input.log](https://github.com/ppy/osu/files/12318230/input.log)
[network.log](https://github.com/ppy/osu/files/12318231/network.log)
[performance.log](https://github.com/ppy/osu/files/12318232/performance.log)
[runtime.log](https://github.com/ppy/osu/files/12318233/runtime.log)

**Comment by peppy** [comment]
5he game should open your browser. Your PC configuration  (browser association) is probably broken.

**Comment by davidak** [comment]
Yes, that seem to be the case. When i click here to open a URL, nothing happens.

![osu_2023-08-11_05-50-30](https://github.com/ppy/osu/assets/91113/05dc7cb8-df6c-45d7-baeb-9498fce93307)

How do you open links on linux? Does it need dependencies like `xdg-open`? The NixOS package might miss them.
